models/oauth2_application_test.go

```diff
@@ -135,6 +135,25 @@ func TestOAuth2Grant_TableName(t *testing.T) {
 	assert.Equal(t, "oauth2_grant", new(OAuth2Grant).TableName())
 }
 
+func TestGetOAuth2GrantsByUserID(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+	result, err := GetOAuth2GrantsByUserID(1)
+	assert.NoError(t, err)
+	assert.Len(t, result, 1)
+	assert.Equal(t, int64(1), result[0].ID)
+	assert.Equal(t, result[0].ApplicationID, result[0].Application.ID)
+
+	result, err = GetOAuth2GrantsByUserID(34134)
+	assert.NoError(t, err)
```

Comment:
Shouldn't this throw an error if the user does not exist?

Reply:
no, since the not existing user has no grants. Error may only be thrown if you want to access exactly one resource.

Reply:
Fair enough.